docs/protocol.md

```diff
@@ -204,6 +209,81 @@ Here are the different methods that can be called:
 * Heartbeat
 * GetMessagesById
 
+### Greeting a server
+🧭 **RPC Message** > **Query** > **Greet Server**
+
+By sending a greet server message, a server can transmit its public key as well as the 
+addresses of its client and server endpoints to another server.
+
+This message should be sent by the server that opens a connection with another server. It should trigger
+a `GreetServer` message from the server that receives the connection containing its own information. 
```

Comment:
Where is this message sent (i.e. on what channel, the default one?)

Reply:
It's not sent to any channel similarly to the `heartbeat `and `getMessagesById` messages.

Reply:
Yes, that was me being dumb sorry!